### PR TITLE
Change wrapper attributes to strings

### PR DESCRIPTION
**Pull request by kjnsn**
Not all states can be captured with `bool`. Specifically followAdditionalWrappers has a default of true, but this cannot be detected with a boolean. For example, having the attribute not present and having the attribute with the value "false" will both result in the parsing returning a boolean with value false.

### CODE REVIEW

File: duration_test.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ func TestDurationUnmarshal(t *testing.T) {
 		assert.Equal(t, Duration(2*time.Second), d)
 	}
 	d = 0
+	if assert.NoError(t, d.UnmarshalText([]byte(" 00:00:02 "))) {
+		assert.Equal(t, Duration(2*time.Second), d)
+	}
```
Comment (kalbasit) [inline duration_test.go]: can you please move this to a different PR?

File: vast.go [hunk 2]
```diff
@@ -127,9 +127,9 @@ type Wrapper struct {
 	// xml element within the Extensions element.
 	Extensions []Extension `xml:"Extensions>Extension,omitempty"`
 
-	FallbackOnNoAd           bool `xml:"fallbackOnNoAd,attr,omitempty"`
-	AllowMultipleAds         bool `xml:"allowMultipleAds,attr,omitempty"`
-	FollowAdditionalWrappers bool `xml:"followAdditionalWrappers,attr,omitempty"`
+	FallbackOnNoAd           string `xml:"fallbackOnNoAd,attr,omitempty"`
+	AllowMultipleAds         string `xml:"allowMultipleAds,attr,omitempty"`
+	FollowAdditionalWrappers string `xml:"followAdditionalWrappers,attr,omitempty"`
```
Comment (kalbasit) [inline vast.go]: this looks good, @rs please merge once the above is moved out to a different PR.

File: duration_test.go (hunk 1 above)
Comment (rs) [reply duration_test.go]: That would be better yes.

**Comment by rs** [comment]
Wouldn't using a `bool` pointer be a better solution to capture the 3 states?
